modules/pacing/paced_sender.cc

```diff
@@ -18,6 +18,11 @@
 #include <vector>
 
 #include "absl/memory/memory.h"
+
+#ifdef INTEL_TELEMETRY
```

Comment:
INTEL or OWT?

Reply:
INTEL.   That's to enable telemetry for endgame, and is using EndGame's telemetry module.